### PR TITLE
Issue #1583 Fix localisation issue for active libvirt networks

### LINKED ISSUE #1583: minishift starts fails on Fedora with German locale
minishift 1.7.0
fedora 26 

Minishift does not start with german locale, when I change to "en_US:UTF-8" it works. 

```
[hygl@localhost ~]$ virsh net-list --all
 Name                 Status     Automatischer Start Bleibend
----------------------------------------------------------

[hygl@localhost ~]$ sudo virsh net-list --all
 Name                 Status     Automatischer Start Bleibend
----------------------------------------------------------
 default              Aktiv      ja            ja

[hygl@localhost ~]$ groups
hygl wheel libvirt docker
[hygl@localhost ~]$ minishift start --metrics
-- Checking if KVM driver is installed ... 
   Driver is available at /usr/local/bin/docker-machine-driver-kvm ... 
   Checking driver binary is executable ... OK
-- Checking if Libvirt is installed ... OK
-- Checking if Libvirt default network is present and active ... FAIL
   See the 'Setting Up the Driver Plug-in' topic for more information`
```

### PR DESCRIPTION
**Pull request by gbraad**
Fixes #1583

### CODE REVIEW

File: cmd/minishift/cmd/start_preflight.go [hunk 1]
```diff
@@ -263,9 +270,33 @@ func checkLibvirtInstalled() bool {
 	return true
 }
 
-//checkLibvirtDefaultNetwork returns true if the "default" network is present and active
-func checkLibvirtDefaultNetwork() bool {
+//checkLibvirtDefaultNetworkExists returns true if the "default" network is present
+func checkLibvirtDefaultNetworkExists() bool {
+	cmd := exec.Command("virsh", "--connect", "qemu:///system", "net-list")
```
Comment (LalatenduMohanty) [inline cmd/minishift/cmd/start_preflight.go]: @gbraad Shouldn't it be `LC_ALL=en_us.UTF8 virsh --connect qemu:///system net-list`?

File: cmd/minishift/cmd/util/env.go [hunk 2]
```diff
@@ -32,6 +34,19 @@ var (
 	BooleanNoValueError = errors.New("No value given.")
 )
 
+func ReplaceEnv(variables []string, varName string, value string) []string {
```
Comment (LalatenduMohanty) [inline cmd/minishift/cmd/util/env.go]: Please add the description of the function (as comment)

File: cmd/minishift/cmd/util/env.go [hunk 3]
```diff
@@ -32,6 +34,19 @@ var (
...
+	for _, e := range variables {
+		pair := strings.Split(e, "=")
+		if pair[0] != varName {
+			result = append(result, e)
+		}
+	}
+	result = append(result, fmt.Sprintf("%s=%s", varName, value))
+
+	return result
+}
```
Comment (LalatenduMohanty) [inline cmd/minishift/cmd/util/env.go]: We should add an unit test for this function.

File: cmd/minishift/cmd/start_preflight.go (hunk 1 above)
Comment (gbraad) [reply cmd/minishift/cmd/start_preflight.go]: That will result in `LC_ALL=en_US.UTF8 is not a command`. This is not a shell call, but just a process invocation.

**Comment by gbraad** [comment]
```
 gbraad  …  github.com  minishift  minishift  export LC_ALL=de_DE.utf8
 gbraad  …  github.com  minishift  minishift  virsh --connect qemu:///system net-list
 Name                 Status     Automatischer Start Bleibend
----------------------------------------------------------
 default              Aktiv      ja            ja
 docker-machines      Aktiv      ja            ja

 gbraad  …  github.com  minishift  minishift  minishift start           ST 9   issue-1583 
-- Checking if KVM driver is installed ... 
   Driver is available at /usr/local/bin/docker-machine-driver-kvm ... 
   Checking driver binary is executable ... OK
-- Checking if Libvirt is installed ... OK
-- Checking if Libvirt default network is present ... OK
-- Checking if Libvirt default network is active ... OK
-- Checking the ISO URL ... OK
-- Starting local OpenShift cluster using 'kvm' hypervisor ...
-- Minishift VM will be configured with ...
   Memory:    4 GB
   vCPUs :    2
   Disk size: 8 GB
-- Starting Minishift VM .....^C
```

**Comment by gbraad** [comment]
Failure in CentOS CI is due to a documentation issue:

See: https://github.com/minishift/minishift/issues/1651, https://github.com/openshift/origin/issues/17165

**Comment by LalatenduMohanty** [comment]
retest this please.

**Pull request merged**